utils/api.py

```diff
@@ -47,9 +47,12 @@ def _result_processor(result):
         if not isinstance(result, dict):
             return result
         if "error" in result:
-            # raise
-            raise APIException(
-                "{}: {}".format(result["error"]["klass"], result["error"]["message"]))
+            if isinstance(result['error'], dict):
```

Comment:
No idea, I just saw that in logs and never saw that before.

*/me does not like the cfme rest api much*